semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -211,7 +211,9 @@ def setup_parser():
     parser.add_argument(
         "--only",
         choices=list(CONVENTION_CLS_BY_GROUP_TYPE.keys()),
-        help="Process only semantic conventions of the specified type.",
+        type=str,
+        nargs="+",
```

Comment:
I think `+` is usually a bad idea since it may eat too many arguments. Maybe a manual split by `,` would be better, or the same method we use for codegeration's `-D`, method=append https://github.com/lmolkova/build-tools/blob/alt-fix-codegen-119/semantic-conventions/src/opentelemetry/semconv/main.py#L139. But my argparse skills are a bit rusty, so not sure maybe it is OK.

Reply:
Did comma-separated string.  `--only span --only event` look weird and I'd need to find a better alias.